fabrication.py

```diff
@@ -393,3 +393,25 @@ def generate_bom(self):
         self.logger.info(
             "Finished generating BOM file %s", os.path.join(self.outputdir, bomname)
         )
+
+    def check_plausibility(self) -> str:
+        """Check the plausibility of the parts, there should be just one value per LCSC number."""
```

Comment:
can we document what the return value means? I presume its an empty string if no issues were found but requires reviewing code to determine that.